python-rhsm/python-rhsm.spec

```diff
@@ -30,36 +20,52 @@ Source0: %{name}-%{version}.tar.gz
 URL: http://www.candlepinproject.org
 BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 
+%if 0%{?suse_version}
+BuildRequires: python-devel >= 2.6
+%else
+BuildRequires: python2-devel
+%endif
+BuildRequires: python-setuptools
+BuildRequires: python-six
+BuildRequires: openssl-devel
+
 %if %use_m2crypto
 %if 0%{?suse_version}
 Requires: python-m2crypto
 %else
 Requires: m2crypto
 %endif
 %endif
+Requires: python-dateutil
 Requires: python-iniparse
+# rpm-python is an old name for python2-rpm but RHEL6 uses the old name
 Requires: rpm-python
-Requires: python-dateutil
-%if %use_simplejson
-Requires: python-simplejson
-%endif
-Requires: python-rhsm-certificates = %{version}-%{release}
 Requires: python-six
+Requires: python-rhsm-certificates = %{version}-%{release}
 
-%if 0%{?suse_version}
-BuildRequires: python-devel >= 2.6
-%else
-BuildRequires: python2-devel
-%endif
-BuildRequires: python-setuptools
+%global _description \
+A small library for communicating with the REST interface of a Red Hat Unified\
+Entitlement Platform. This interface is used for the management of system\
+entitlements, certificates, and access to content.
+
+%description %{_description}
+
+
+%package -n python3-%{name}
```

Comment:
Similar to how we conditionally build ex. initial-setup plugins, let's have conditionals for the Python 3 packages.